Makefile

```diff
@@ -0,0 +1,25 @@
+init:
+	[ -f .env ] || cp .env.example .env
+	make run
+
+run:
+	docker compose up -d
+	docker logs -f e-motion-web
```

Comment:
```suggestion
```

Reply:
I think it is actually needed here because otherwise you wouldn't know the ip address. With Kamil's suggestions the app ip is not localhost, but somewhere in the 172.2.0.0/16 subnet. I can also add something like ```make logs``` but i don't think it is nescessary. What do you think?